db/migrate/20200410000000_create_initial_entities.rb

```diff
@@ -0,0 +1,78 @@
+class CreateInitialEntities < ActiveRecord::Migration[6.0]
+  def change
+    create_table(:items) do |table|
+      table.decimal(:added_sugar)
+      table.decimal(:calcium)
+      table.decimal(:calories)
+      table.decimal(:carbs)
+      table.decimal(:cholesterol)
+      table.decimal(:fat)
+      table.decimal(:fiber)
+      table.decimal(:grams_per_tablespoon)
+      table.decimal(:iron)
+      table.decimal(:magnesium)
+      table.decimal(:monounsaturated_fat)
+      table.string(:name, null: false)
+      table.decimal(:niacin)
+      table.decimal(:phosphorus)
+      table.decimal(:polyunsaturated_fat)
+      table.decimal(:potassium)
+      table.decimal(:protein)
+      table.decimal(:riboflavin)
+      table.decimal(:saturated_fat)
+      table.decimal(:sodium)
+      table.decimal(:sugar)
+      table.decimal(:thiamin)
+      table.decimal(:trans_fat)
+      table.string(:upc)
+      table.decimal(:vitamin_a)
+      table.decimal(:vitamin_b6)
+      table.decimal(:vitamin_b12)
+      table.decimal(:vitamin_c)
+      table.decimal(:vitamin_d)
+      table.decimal(:vitamin_e)
+      table.decimal(:vitamin_k)
+      table.decimal(:zinc)
+      table.timestamps
+      table.index(%i[name])
```

Comment:
This isn't going to help us since we won't be doing searches for exact values. A fulltext index is the only useful index to use here.